src/controllers/profile-settings.controller.js

```diff
@@ -0,0 +1,64 @@
+import ResponseService from '../services/response.service';
+import UserService from '../services/user.service';
+import JwtService from '../services/jwt.service';
+/**
+ *
+ *
+ * @class ProfileSettingsController
+ */
+class ProfileSettingsController {
```

Comment:
You should use class name  convention that Temitayo recommended

Reply:
@gadishimwe for as @adesege said:
* `method names must contain a verb` Not class names.
* `Class names must match with their equivalent import and exports`

I think the above line is correct. No need for change. Thx